### ISSUE TITLE
platform.js causes page to be blocked by ad blockers

### ISSUE THREAD

**Issue by davidwagner** [Cal-CS-61A-Staff/cs61a-apps#73]
Copying from https://github.com/Cal-CS-61A-Staff/examtool-web/issues/14:  https://exam.cs61.org/ loads https://apis.google.com/js/platform.js, which is blocked by some ad blockers, causing the page to not load.  I had multiple students in Data 8 report this issue.  It would be useful to load that script from elsewhere, so the page isn't blocked by ad blockers.

**Comment by chrononyan** [comment]
I don't think this will be effective in the long run -- if filter lists decide to block this script (rightfully for social functionality), they may eventually block all re-hosted instances of the script as they are discovered. Most adblockers allow users to make exceptions for pages they trust, and circumventing this model doesn't seem ideal.

IMO the better way is to implement server-side Google OAuth2 login, which removes this dependency entirely, but is somewhat more headache to deal with (Google OAuth client rules...)
